src/main/java/com/pscheduler/server/core/DatabaseLoader.java

```diff
@@ -45,7 +48,17 @@ public void run(ApplicationArguments args) throws Exception {
         }
         courses.save(courseList);
 
-        CourseGPA course = new CourseGPA(
+
+        GradeParser grader = new GradeParser();
+
+        // REMEMBER TO CHANGE THIS TO PARSEALLFILES

```

Comment:
Remove this comment for now, we can change it later

Reply:
Removed